internal/uvm/create_wcow.go

```diff
@@ -194,7 +194,12 @@ func CreateWCOW(ctx context.Context, opts *OptionsWCOW) (_ *UtilityVM, err error
 		}
 	}
 
-	uvm.scsiLocations[0][0].hostPath = doc.VirtualMachine.Devices.Scsi["0"].Attachments["0"].Path
+	// Fairly certain there won't be any Scsi mounts allocated before a call to this but just to be sure
+	if uvm.scsiLocations[0][0] != nil {
+		uvm.scsiLocations[0][0].HostPath = doc.VirtualMachine.Devices.Scsi["0"].Attachments["0"].Path
+	} else {
+		uvm.scsiLocations[0][0] = &ScsiMount{HostPath: doc.VirtualMachine.Devices.Scsi["0"].Attachments["0"].Path}
+	}
```

Comment:
why can we not just do 
`uvm.scsiLocations[0][0] = &ScsiMount{HostPath: doc.VirtualMachine.Devices.Scsi["0"].Attachments["0"].Path}`
in both cases?

Reply:
Are you planning on doing this or did you find an issue?

Reply:
Had some questions regarding this but @kevpar cleared them up so will be going forward with your suggestion just with the uvm made above embedded also.